dev/integration_tests/ios_host_app/Host/AppDelegate.m

```diff
@@ -3,6 +3,51 @@
 // found in the LICENSE file.
 
 #import "AppDelegate.h"
+#import "MainViewController.h"
```

Comment:
This was moved from [dev/integration_tests/ios_add2app/ios_add2app/AppDelegate.m](https://github.com/flutter/flutter/pull/61007/files#diff-9e149ff1c1dcf4b51edd1f16e15eea54) but git didn't follow the rename.